### PR TITLE
New package: zellij-0.25.0

### LINKED ISSUE #35686: Package request: zellij
A terminal workspace with batteries included, similar to tmux

https://github.com/zellij-org/zellij

### PR DESCRIPTION
**Pull request by tranzystorek-io**
Closes #35686

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

**Comment by cinerea0** [comment]
zellij's dependency on wasmer means the architectures this can be packaged for are limited, at least according to this older PR: https://github.com/void-linux/void-packages/pull/32990.

**Comment by tranzystorek-io** [comment]
According to https://github.com/google/cargo-raze/issues/114#issuecomment-547080315, on the failing targets the `autocfg` build script for `indexmap` cannot detect that `std` is available and ends up compiling an `IndexSet` without a default build hasher. I'm wondering if we can manually specify `--cfg=has_std` somehow.

**Comment by paper42** [comment]
> According to [google/cargo-raze#114 (comment)](https://github.com/google/cargo-raze/issues/114#issuecomment-547080315), on the failing targets the `autocfg` build script for `indexmap` cannot detect that `std` is available and ends up compiling an `IndexSet` without a default build hasher. I'm wondering if we can manually specify `--cfg=has_std` somehow.

#34889

**Comment by tranzystorek-io** [comment]
> Is this similar to #34889?

Yes, it seems to be exactly that issue, though it's not directly in zellij code but in `gimli` 0.22

File: srcpkgs/zellij/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'zellij'
+pkgname=zellij
+version=0.25.0
+revision=1
+# Dependency crate `wasmer-vm` only supports the following archs: x86_64, arm64, x86
+# https://docs.wasmer.io/ecosystem/wasmer/wasmer-features#compiler-support-by-chipset
+archs="x86_64* aarch64* i686"
```
Comment (paper42) [inline srcpkgs/zellij/template:7]:
```suggestion
archs="x86_64* aarch64* i686*"
```

**Pull request merged**